modules/ols-creating-lightspeed-custom-resource-file-using-cli.adoc

```diff
@@ -53,19 +53,20 @@ spec:
   llm:
     providers:
     - credentialsSecretRef:
-        name: openai-api-keys
+        name: rhelai-api-keys <1>
       models:
       - name: models/<model_name>
       name: rhelai
       type: rhelai_vllm
-      url: <URL> <1>
+      url: <URL> <2>
```

Comment:
```suggestion
      url: <url> <2>
```

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values

Reply:
done